superset-frontend/package-lock.json

```diff
@@ -20532,7 +20532,8 @@
         },
         "kind-of": {
           "version": "6.0.2",
-          "resolved": "",
+          "resolved": "https://registry.npmjs.org/kind-of/-/kind-of-6.0.2.tgz",
```

Comment:
are these changes intended?

Reply:
Not really

Reply:
This stuff always seems to alternate back and forth. Not sure if there's a good way to reduce churn on this.